mappings/net/minecraft/client/gui/screen/ingame/StonecutterScreen.mapping

```diff
@@ -4,6 +4,15 @@ CLASS net/minecraft/class_3979 net/minecraft/client/gui/screen/ingame/Stonecutte
 	FIELD field_17672 canCraft Z
 	FIELD field_17673 TEXTURE Lnet/minecraft/class_2960;
 	FIELD field_17674 scrollAmount F
+	FIELD field_32372 SCROLLBAR_WIDTH I
+	FIELD field_32373 SCROLLBAR_HEIGHT I
+	FIELD field_32374 RECIPE_LIST_COLUMNS I
+	FIELD field_32375 RECIPE_LIST_ROWS I
+	FIELD field_32376 RECIPE_ENTRY_WIDTH I
+	FIELD field_32377 RECIPE_ENTRY_HEIGHT I
+	FIELD field_32378 SCROLLBAR_AREA_HEIGHT I
+	FIELD field_32379 RECIPE_LIST_X_OFFSET I
+	FIELD field_32380 RECIPE_LIST_Y_OFFSET I
```

Comment:
```suggestion
	FIELD field_32379 RECIPE_LIST_OFFSET_X I
	FIELD field_32380 RECIPE_LIST_OFFSET_Y I
```
Yarn conventions: "Coordinates can be named `x`, `y`, and `z` when it's clear what they represent. If clarification is needed, add a word in front of the coordinate (`velocityX`, not `xVelocity`)."